docs/source/quick_start.rst

```diff
@@ -0,0 +1,175 @@
+    ==================================================
+    Expanding step 'run-lulesh'
+    ==================================================
+    -------- Used Parameters --------
+    set(['SIZE', 'ITERATIONS'])
+    ---------------------------------
+    2018-08-07 00:58:57,308 - maestrowf.datastructures.core.study:_stage_parameterized:578 - INFO -
+    **********************************
+    Combo [SIZE.10.TRIAL.1.ITER.10]
+    **********************************
+    2018-08-07 00:58:57,309 - maestrowf.datastructures.core.study:_stage_parameterized:599 - INFO - Searching for workspaces...
+    cmd = /home/travis/build/LLNL/maestrowf/testing/lulesh/LULESH/lulesh2.0 -s 10 -i 10 -p > SIZE.10.ITER.10.log
+    2018-08-07 00:58:57,309 - maestrowf.datastructures.core.study:_stage_parameterized:630 - INFO - New cmd = /home/travis/build/LLNL/maestrowf/testing/lulesh/LULESH/lulesh2.0 -s 10 -i 10 -p > SIZE.10.ITER.10.log
+    2018-08-07 00:58:57,309 - maestrowf.datastructures.core.study:_stage_parameterized:640 - INFO - Processing regular dependencies.
+    2018-08-07 00:58:57,309 - maestrowf.datastructures.core.study:_stage_parameterized:648 - INFO - Adding edge (make-lulesh, run-lulesh_ITER.10.SIZE.10)...
+    2018-08-07 00:58:57,309 - maestrowf.datastructures.core.study:_stage_parameterized:578 - INFO -
+    **********************************
+    Combo [SIZE.10.TRIAL.2.ITER.20]
+    **********************************
+    2018-08-07 00:58:57,309 - maestrowf.datastructures.core.study:_stage_parameterized:599 - INFO - Searching for workspaces...
+    cmd = /home/travis/build/LLNL/maestrowf/testing/lulesh/LULESH/lulesh2.0 -s 10 -i 20 -p > SIZE.10.ITER.20.log
+    2018-08-07 00:58:57,309 - maestrowf.datastructures.core.study:_stage_parameterized:630 - INFO - New cmd = /home/travis/build/LLNL/maestrowf/testing/lulesh/LULESH/lulesh2.0 -s 10 -i 20 -p > SIZE.10.ITER.20.log
+    2018-08-07 00:58:57,309 - maestrowf.datastructures.core.study:_stage_parameterized:640 - INFO - Processing regular dependencies.
+    2018-08-07 00:58:57,310 - maestrowf.datastructures.core.study:_stage_parameterized:648 - INFO - Adding edge (make-lulesh, run-lulesh_ITER.20.SIZE.10)...
+
+Once expansion is complete, Maestro will prompt you to confirm if you'd like to launch the study. Simply confirm with a `y` and hit enter.::
+
+    $ Would you like to launch the study? [yn] y
+
+Maestro will launch a conductor in the background using ``nohup`` in order to monitor the executing study.
+
+
+Monitoring a Running Study
+***************************
+
+Once the conductor is spun up, you will be retuned to the command line prompt. There should now be a the ``.tests/lulesh`` within the root of the repository. This directory represents the executing study's workspace, or where Maestro will place this study's data, logs, and state. For a more in-depth description of the contents of a workspace see the documentation about :doc:`Study Workspaces <./maestro_organization>`.

```

Comment:
This should be: "returned to the command line prompt".

Reply:
Makes sense -- done.